app/models/Email.scala

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
What about adding a package `email` and splitting this file in 4 new files in the new package?